packages/components/tag/src/tag.css

```diff
@@ -0,0 +1,238 @@
+	outline-color: var(--ts-color-red-dark);
+}
+:host([type='blue']:focus-within) {
+	outline-color: var(--ts-color-blue-dark);
+}
+:host([type='blue-lite']:focus-within) {
+	outline-color: var(--ts-color-blue-light);
+}
+:host(:not([type]):focus-within) {
+	outline-color: var(--ts-color-gray-light);
+}
+
+/* Labels and values (text content) .................................*/
+
+.label {
+	text-transform: uppercase;
+	font-weight: var(--ts-fontweight-semibold);
+}
+
+.label,
+.value {
+	white-space: nowrap;
+	overflow: hidden;
+	text-overflow: ellipsis;
+}
+
+.value {
+	flex: 1;
+}
+
+/* Colors ....................................................*/
+
+:host([type='success']),
+:host([type='warning']),
+:host([type='danger']),
+:host([type='blue']) {
+	& > .container {
+		color: var(--ts-color-white);
+	}
+}
+
+:host(:not([busy]):not([locked])[type='success']) {
+	& .container {
+		background-color: var(--ts-color-green);
+	}
+}
+
+:host(:not([busy]):not([locked])[type='success'][clickable]) {
+	& .container:hover {
+		background-color: var(--ts-color-green-dark);
+	}
+}
+
+:host(:not([busy]):not([locked])[type='warning']) {
+	& .container {
+		background-color: var(--ts-color-orange);
+	}
+}
+
+:host(:not([busy]):not([locked])[type='warning'][clickable]) {
+	& .container:hover {
+		background-color: var(--ts-color-orange-dark);
+	}
+}
+
+:host(:not([busy]):not([locked])[type='warning-lite']) {
+	& .container {
+		background-color: var(--ts-color-orange-lightest);
+	}
+}
+
+:host(:not([busy]):not([locked])[type='warning-lite'][clickable]) {
+	& .container:hover {
+		background-color: var(--ts-color-orange-lighter);
+	}
+}
+
+:host(:not([busy]):not([locked])[type='danger']) {
+	& .container {
+		background-color: var(--ts-color-red);
+	}
+}
+
+:host(:not([busy]):not([locked])[type='danger'][clickable]) {
+	& .container:hover {
+		background-color: var(--ts-color-red-dark);
+	}
+}
+
+:host(:not([busy]):not([locked])[type='blue']) {
+	& .container {
+		background-color: var(--ts-color-blue);
+	}
+}
+
+:host(:not([busy]):not([locked])[type='blue'][clickable]) {
+	& .container:hover {
+		background-color: var(--ts-color-blue-dark);
+	}
+}
+
+:host(:not([busy]):not([locked])[type='blue-lite']) {
+	& .container {
+		background-color: var(--ts-color-blue-lighter);
+		& .label,
+		& .value {
+			color: var(--ts-color-blue-dark);
+		}
+	}
+}
+
+:host(:not([busy]):not([locked])[type='blue-lite'][clickable]) {
+	& .container:hover {
+		background-color: var(--ts-color-blue-light);
+	}
+}
+
+:host([locked]) .container {
+	color: var(--ts-color-slate-lightest);
+}
+
+/* Specific clickable styles .....................................................*/
+
+:host([clickable]) .container {
+	cursor: pointer;
+}
+
+:host(:not([type])[clickable]) .container {
+	color: var(--ts-color-blue);
+}
+
+/* Icons ........................................................*/
+
+:host([locked]),
+:host([deletable]) {
+	& .container[dir='ltr'] {
```

Comment:
nit, Generally we have default state, which would be ltr and add specific dir='rtl' overrides.